setup.py

```diff
@@ -2,9 +2,13 @@
 from setuptools.command.test import test as TestCommand
 from setuptools.command.install_scripts import install_scripts
 from setuptools.command.install import install as _install
-import os, sys
+import os, platform, sys
 
 
+PYTHON_37_ERROR = 'Sorry, BiblioPixel does not yet support Python 3.7'
```

Comment:
Maybe this instead? 'Sorry, some of BiblioPixel's dependencies do not yet support Python 3.7. Please use Python 3.6 instead.'
Again, this should only really be an issue with Windows and I guess Mac or does that ship with py3 now?